src/Ast/PhpVersion.php

```diff
@@ -89,6 +89,16 @@ public function isMixedTypehintSupported(): bool
         return $this->isVersionGreaterThanOrEqualTo80();
     }
 
+    public function isEnumSupported(): bool
+    {
+        return $this->isVersionGreaterThanOrEqualTo81();
+    }
+
+    public function isReadonlyPropertiesSupported(): bool
```

Comment:
Maybe here would be more proper name, to user `are` or use `PropertySupported` with `is` prefix. But as we are using `single` versions here in this file, I suggest to change to `isReadonlyPropertySupported` to fit with others.

Reply:
makes sense, fixed